apps/freeablo/farender/spritecache.cpp

```diff
@@ -257,6 +257,9 @@ namespace FARender
 
     void SpriteCache::evict()
     {
+        if (!Render::SpriteGroup::canDelete())
+            return;
```

Comment:
I think, we can get rid of the whole concept of a Sprite Cache. 
I woule be happy with just preloading all the textures we're going to use into the atlas at startup, and if we have problems with this later, then we can figure it out later.

Reply:
Ahh yep sweet, will give this a go.
That method name is also terrible, was meant to be `canDeleteSprites`

Reply:
We could dump a list, by running the game and tracing each load. Or, honestly, we could just preload every CEL/CL2 file in DIABDAT.MPQ.

Reply:
@wheybags I had a go at preloading all the sprites, it seems there are simply too many.
My debug outputs: 1402 Cel files, 110247 individual sprites.
The 2D bin packer mustn't have a very good Big O and gets slower and slower, but even disabling that just the Cel decoding takes > 60s.

I don't see dumping a list of _used_ Cel files as being very maintainable.
Might have to stick with loading files runtime?

Also I had to disable decoding these 3 Cel files as they cause a crash when decoding (they also crash if clicked in the celview executable):
```
"monsters/fireman/firema.cl2"
"monsters/unrav/unravw.cel"
"monsters/darkmage/dmagew.cl2"
```

Reply:
What I would do here is keep some list of sprites, and on first run, decode them all into atlases, and save the atlases as pngs in soem temp folder. Then we can just laod the single pngs on subsequent runs. OFC we would need to invalidate that if the sprite list changes.